danger-kotlin-library/src/test/kotlin/systems/danger/kotlin/GitKtxTest.kt

```diff
@@ -0,0 +1,77 @@
+package systems.danger.kotlin
+
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import systems.danger.kotlin.shell.ShellExecutor
+import systems.danger.kotlin.shell.ShellExecutorFactory
+
+/**
+ * Tests for [Git] extensions
+ */
+internal class GitKtxTest {
+
+    class MockShellExecutor : ShellExecutor {
```

Comment:
what about using a mock library? i was planning to add mockk.. let's use that

Reply:
ok, I personally don't use mock libraries, I prefer fakes, but in this context it's probably better.

Reply:
fd732612fd93224d24b0b63afef7bacca1470e4f